cgi-bin/DW/Console/Command/RenameOpts.pm

```diff
@@ -65,7 +66,7 @@ sub execute {
 
         if ( $cmd eq 'break_redirect' )  {
             if ( $u->break_redirects ) {
-                $u->set_deleted;
+                $u->set_expunged;
```

Comment:
This only gets used on type R accounts right? I.e., the "fake" accounts that currently redirect to people. If so, I _think_ this is safe technically... it tickles my feelers and makes me worry slightly, but I think I've convinced myself it's OK technically.